samples/ios/app/glean-sample-app/ViewController.swift

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Glean
+import glean_sample_framework
```

Comment:
Hm, naming. :D

Reply:
Yeah, don't use dashes in a XCode project...   will try and fix it if it doesn't break everything.